src/core/Object3D.js

```diff
@@ -40,20 +40,24 @@ THREE.Object3D = function () {
 	Object.defineProperties( this, {
 		position: {
 			enumerable: true,
+			configurable: true,
```

Comment:
Yes, `configurable` is needed so that it can be deleted.  You're right, it probably needs configurable after redefining as well - I'll make that change.  It's probably unlikely someone will be rebinding a second time, but best to be consistent.